shared/src/main/scala/mlscript/JSBackend.scala

```diff
@@ -76,9 +76,9 @@ abstract class JSBackend(allowUnresolvedSymbols: Bool) {
       translatePattern(base)
     case Inst(bod) => translatePattern(bod)
     case Ann(ann, receiver) => translatePattern(receiver)
-    case _: Lam | _: App | _: Sel | _: Let | _: Blk | _: Bind | _: Test | _: With | _: CaseOf
-        | _: Subs | _: Assign | _: If | _: New  | NuNew(_) | _: Splc | _: Forall | _: Where
-        | _: Super | _: Eqn | _: AdtMatchWith | _: Rft | _: While =>
+    case _: Lam | _: App | _: Sel | _: Let | _: Blk | _: Bind | _: Test | _: With | _: CaseOf | _: Subs | _: Assign
+        | If(_, _) | New(_, _)  | NuNew(_) | _: Splc | _: Forall | _: Where | _: Super | _: Eqn | _: AdtMatchWith
+        | _: Rft | _: While | _: Quoted | _: Unquoted =>
```

Comment:
What happened here? Why all these unnecessary changes?

Note: I always prefer writing things like `_: If` instead of `If(_, _)` as the former will need to be adapted if/when `If` gets a new argument.